controllers/configentry_controller_ent_test.go

```diff
@@ -405,3 +406,394 @@ func TestConfigEntryController_deletesConfigEntry_consulNamespaces(tt *testing.T
 		})
 	}
 }
+
+func TestConfigEntryController_createsGlobalConfigEntry_consulNamespaces(tt *testing.T) {
```

Comment:
😢 copying all these test is kinda brutal too. I've also played around with a way to reduce the duplication but nothing works great.

Reply:
What do you think about following kinda the way the non-ent controller tests are with some lambdas that can return either the service defaults or the proxy defaults controller?

Reply:
yeah 😞